src/Neo.CLI/Settings.cs

```diff
@@ -93,26 +93,26 @@ public class P2PSettings
 
         public P2PSettings(IConfigurationSection section)
         {
-            this.Port = section.GetValue<ushort>("Port", 10333);
-            this.MinDesiredConnections = section.GetValue("MinDesiredConnections", Peer.DefaultMinDesiredConnections);
-            this.MaxConnections = section.GetValue("MaxConnections", Peer.DefaultMaxConnections);
-            this.MaxConnectionsPerAddress = section.GetValue("MaxConnectionsPerAddress", 3);
+            Port = section.GetValue<ushort>(nameof(Port), 10333);
+            MinDesiredConnections = section.GetValue(nameof(MinDesiredConnections), Peer.DefaultMinDesiredConnections);
+            MaxConnections = section.GetValue(nameof(MaxConnections), Peer.DefaultMaxConnections);
+            MaxConnectionsPerAddress = section.GetValue(nameof(MaxConnectionsPerAddress), 3);
         }
     }
 
     public class UnlockWalletSettings
     {
-        public string? Path { get; }
-        public string? Password { get; }
-        public bool IsActive { get; }
+        public string Path { get; } = string.Empty;
+        public string Password { get; } = string.Empty;
+        public bool IsActive { get; } = false;
 
         public UnlockWalletSettings(IConfigurationSection section)
         {
             if (section.Exists())
             {
-                this.Path = section.GetValue("Path", "");
-                this.Password = section.GetValue("Password", "");
-                this.IsActive = bool.Parse(section.GetValue("IsActive", "false")!);
+                Path = section.GetValue(nameof(Path), string.Empty)!;
```

Comment:
I mean, we should change only the required changes for use dotnet standard

Reply:
Thats what im doing. Believe me, the less work i can do for converting to `dotnet` standard, you bet I'll do it. The reason for that is because your code changes request, and i had some time. Im very O.C.D. when it comes to coding standards and writing code. So I couldn't help myself.

Reply:
> The reason for that is because your code changes request, and i had some time. Im very O.C.D. when it comes to coding standards and writing code.

That's not bad, and I'm not against these changes, but please in a different PR, it helps with tracing and reviewing.

Reply:
But that's like saying, When we want to add something new for example, like `RestServer` we can't implement the whole project. The have to implement pieces of the project's code to keep track of it.

Well I have solution for you. That's why they created `Rebase`.

Reply:
Please open a new PR for these changes, there are no reason to mix things, focus on the description and title of the PR.

Reply:
I checked in the main repository and these null messages doesn't appear, why if it's nullable?